samples/cpp/benchmark_app/statistics_report.cpp

```diff
@@ -119,7 +119,7 @@ void StatisticsReport::dump_sort_performance_counters_request(CsvDumper& dumper,
     dumper.endLine();
 
     for (const auto& layer : perfCounts) {
-        if (status_names[(int)layer.status] == "EXECUTED") {
+        if (strcmp(status_names[(int)layer.status], "EXECUTED") == 0) {
```

Comment:
strcmp maybe bring security risk, it is better use C++ api to compare them.

Reply:
@riverlijunjie https://en.cppreference.com/w/cpp/string/byte/strncmp this is sufficient?

Reply:
yes, strncmp is better than strcmp.